src/components/SignUp.tsx

```diff
@@ -30,28 +28,25 @@ const { SIGN_IN_LINK_TEXT, SIGN_UP_BUTTON, SIGN_UP_HEADER, NAME_FIELD_LABEL } =
 const SignUp = () => {
   const { t } = useAuthTranslation();
 
-  const [email, setEmail] = useState('');
-  const [name, setName] = useState('');
-  const [nameError, setNameError] = useState(false);
+  const [email, setEmail] = useState<string>('');
+  const [name, setName] = useState<string>('');
+  const [nameError, setNameError] = useState<string | null>(null);
   // enable validation after first click
   const [shouldValidate, setShouldValidate] = useState(false);
 
-  const { mutate: signUp } = useMutation(MUTATION_KEYS.SIGN_UP);
+  const { mutate: signUp } = useMutation<any, any, any>(MUTATION_KEYS.SIGN_UP);
```

Comment:
Use this instead of `any` ?
```suggestion
  const { mutate: signUp } = useMutation<unknown, unknown, {email: string; name: string}>(MUTATION_KEYS.SIGN_UP);
```